### DISCUSSION TITLE
Generate playlist question

### DISCUSSION THREAD

**Discussion by Kajirinn** [Tyrrrz/YoutubeExplode#561] [Q&A]
Hi there,

Duplicate of #494, but this trick no longer seems to work. Within youtube itself though, RD still seems to be the main factor in generating custom based playlists for videos. How would one go about achieving this effect with the current framework?

Thank you in advance,
Kajirinn.

**Comment by Kajirinn** [comment]
Does anybody have an idea? I've tried various methods but I'm still stuck with the frustration of an unsolved problem.

**Reply by Tyrrrz** [reply]
I think they changed something to RD playlists, so I'm not sure how they work now

**Reply by Kajirinn** [reply]
Oh okay, I thought I was personally missing something. That's sad to hear, but I assume it can't be helped; thank you!
